sway-core/src/asm_generation/fuel/fuel_asm_builder.rs

```diff
@@ -131,7 +131,7 @@ impl<'ir, 'eng> FuelAsmBuilder<'ir, 'eng> {
             self.non_entries
                 .clone()
                 .into_iter()
-                .map(|ops| AbstractInstructionSet { ops })
+                .map(|ops| (AbstractInstructionSet { ops }))
```

Comment:
Is this really necessary?

Reply:
Removing it. It was a result of multiple different attempts, and got left out during some later refactor :-)